src/Container/ServiceManager/AutowireFactory.php

```diff
@@ -21,18 +21,19 @@
  */
 class AutowireFactory implements AbstractFactoryInterface
 {
-    /**
-     * @var GenericAutowireFactory
-     */
+    /** @var GenericAutowireFactory */
     private $factory;
 
-    public function __construct(GenericAutowireFactory $factory = null)
+    public function __construct(?GenericAutowireFactory $factory = null)
```

Comment:
Please revert his change as it introduces BC break.

Reply:
those ? are added by cs-fix